docs/docs/using-the-template.md

```diff
@@ -0,0 +1,158 @@
+
+## Check out a branch
+
+We'll talk about code review later, but it's a good practice to use feature branches and pull requests to keep your development organized. Now that you have source control configured, you can check out a branch to work in with:
+
+```
+git checkout -b initial-exploration
+```
+
+## Open a notebook
+
+Now you're ready to do some analysis! Make sure that your project-specific environment is activated (you can check with `which jupyter`) and run `jupyter notebook notebooks` to open a Jupyter notebook in the `notebooks/` folder. You can start by creating a new notebook and doing some exploratory data analysis. We often name notebooks with a scheme that looks like this:
+
+```
+0.01-pjb-data-source-1.ipynb
+```
+
+ - `0.01` - Helps leep work in chronological order. The structure is `PHASE.NOTEBOOK`. `NOTEBOOK` is just the Nth notebook in that phase to be created. For phases of the project, we generally use a scheme like the following, but you are welcome to design your own conventions:
+    - `0` - Data exploration - often just for exploratory work
+    - `1` - Data cleaning and feature creation - often writes data to `data/processed` or `data/interim`
+    - `2` - Visualizations - often writes publication-ready viz to `reports`
+    - `3` - Modeling - training machine learning models
+    - `4` - Publication - Notebooks that get turned directly into reports
+- `pjb` - Your initials; this is helpful for knowing who created the notebook and prevents collisions from people working in the same notebook.
+- `data-source-1` - A description of what the notebook covers 
+
+Now that you have your notebook going, start your analysis!
+
+## Refactoring code into shared modules
+
+As your project goes on, you'll want to refactor your code in a way that makes it easy to share between notebooks and scripts. We recommend creating a module in the `{{ cookiecutter.module_name }}` folder that contains the code you use in your project. This is a good way to make sure that you can use the same code in multiple places without having to copy and paste it.
+
+Because the default structure is a Python package and is installed by default, you can do the following to make that code available to you within a Jupyter notebook.
+
+First, we recommend turning on the `autoreload` extension. This will make Jupyter always go back to the source code for the module rather than caching it in memory. If your notebook isn't reflecting the latest changes from your changes to a `.py` file, try restarting the kernel and make sure `autoreload is on. We add a cell at the top of the notebook with the following:
```

Comment:
```suggestion
First, we recommend turning on the `autoreload` extension. This will make Jupyter always go back to the source code for the module rather than caching it in memory. If your notebook isn't reflecting the latest changes from your changes to a `.py` file, try restarting the kernel and make sure `autoreload` is on. We add a cell at the top of the notebook with the following:
```